README.md

```diff
@@ -97,7 +110,7 @@ openToYearSelection | boolean | false | Open datepicker from year selection
 minDate | date | '1900-01-01' | Minimum selectable date
 maxDate | date | '2100-01-01' | Maximum selectable date
 onChange | func | required | Callback firing when date accepted
-returnMoment | boolean | true | Will return moment object in onChange
+returnMoment | boolean | true | Will return moment object in onChange (if moment utils using)
```

Comment:
let's remove this prop (as proposed in #172) in order to improve API consistence

Reply:
I dont think its a good proposal, because a lot of project is using moment + pickers now and that will cause a lot of breaking changes because it will require all users to process native date.
Or you sugges always return moment for moment pickers and date for datefns?

Reply:
>Or you sugges always return moment for moment pickers and date for datefns

Yes, in case of date-fns `returnMoment` prop doesn't make any sense.

> because a lot of project is using moment + pickers now and that will cause a lot of breaking changes

Well, there will be breaking changes anyway

>because it will require all users to process native date

Yes, but they can use `utils` to do it.

This way we'll have consistent and predictable API. If we'll add utils for one more date processing library in the future - API will not change.

Reply:
You maybe not fully understand the purpose of utils architecture? Its a way to users that work with moment pass moment objects and get moment objects from the pickers, and also not reduce bundle size by another date library. 
So it just the best of utils - return moment if moment utils and date if date utils

Reply:
Does `returnMoment` still make sense?

Reply:
Maybe I will remove it and force passing moment everywhere if moment-utils provided